### PR TITLE
[Forwardport] Fix method name (typo)

### PR DESCRIPTION
**Pull request by vijay-wagento**
### Original Pull Request 
 https://github.com/magento/magento2/pull/15514
### Description
This PR deprecates a current method with a typo and introduces the new method with correct spelling

### Fixed Issues (if relevant)
none

### Contribution checklist
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds on Travis CI are green)

### CODE REVIEW

File: app/code/Magento/Multishipping/Block/Checkout/AbstractMultishipping.php [hunk 1]
```diff
@@ -5,7 +5,7 @@
  */
 
 /**
- * Multishipping checkout base abstract block
+ * Mustishipping checkout base abstract block
```
Comment (orlangur) [inline app/code/Magento/Multishipping/Block/Checkout/AbstractMultishipping.php]: Mustishipping?
Cc: @avoelkl @dmanners

**Review by orlangur — CHANGES_REQUESTED** [review]

File: app/code/Magento/Multishipping/Block/Checkout/AbstractMultishipping.php (hunk 1 above)
Comment (dmanners) [reply app/code/Magento/Multishipping/Block/Checkout/AbstractMultishipping.php]: Hi @orlangur thanks for this. I have already reached out to @avoelkl with regards to this.

Comment (avoelkl) [reply app/code/Magento/Multishipping/Block/Checkout/AbstractMultishipping.php]: I would rather not back/forwardort this PR as i accendentially added (and reverted later again) a commit that was supposed to be in another PR.
The typos were fixed in a seperate PR.

Comment (VladimirZaets) [reply app/code/Magento/Multishipping/Block/Checkout/AbstractMultishipping.php]: @vijay-wagento @avoelkl @dmanners 
I think we should fix typos in current PR. We haven't the reason to do it in separate PR.

Comment (vijay-wagento) [reply app/code/Magento/Multishipping/Block/Checkout/AbstractMultishipping.php]: @VladimirZaets I will do it in same PR

**Comment by magento-cicd2** [comment]
[CLA assistant check](https://cla.dev.magento.com/magento/magento2?pullRequest=15745) <br/>All committers have signed the CLA.

**Comment by magento-engcom-team** [comment]
Hi @vijay-wagento. Thank you for your contribution. 
Changes from your Pull Request will be available with the upcoming 2.3.0 release.

**Pull request merged**